evacuation/cleanup_test.go

```diff
@@ -163,36 +163,38 @@ var _ = Describe("EvacuationCleanup", func() {
 		})
 
 		Describe("deleting running containers", func() {
-			It("should delete all of the containers that are still running", func() {
-				fakeExecutorClient.ListContainersReturnsOnCall(1,
-					[]executor.Container{
-						{
-							Guid:    "container1",
-							State:   executor.StateRunning,
-							RunInfo: executor.RunInfo{LogConfig: executor.LogConfig{Guid: "log-guid-1", SourceName: "source-name-1", Index: 0}},
-						},
-						{Guid: "container2",
-							State:   executor.StateCompleted,
-							RunInfo: executor.RunInfo{LogConfig: executor.LogConfig{Guid: "log-guid-2", SourceName: "source-name-2", Index: 1}},
+			Context("When containers fail to exit initially", func() {
+				BeforeEach(func() {
+					fakeExecutorClient.ListContainersReturnsOnCall(1,
+						[]executor.Container{
+							{
+								Guid:    "container1",
+								State:   executor.StateRunning,
+								RunInfo: executor.RunInfo{LogConfig: executor.LogConfig{Guid: "log-guid-1", SourceName: "source-name-1", Index: 0}},
+							},
+							{Guid: "container2",
+								State:   executor.StateCompleted,
+								RunInfo: executor.RunInfo{LogConfig: executor.LogConfig{Guid: "log-guid-2", SourceName: "source-name-2", Index: 1}},
+							},
 						},
-					},
-					nil,
-				)
-				fakeExecutorClient.ListContainersReturnsOnCall(2, []executor.Container{}, nil)
-
-				Consistently(errCh).ShouldNot(Receive())
-				fakeClock.Increment(time.Second * 1)
+						nil,
+					)
+					fakeExecutorClient.ListContainersReturnsOnCall(3, []executor.Container{}, nil)
```

Comment:
Any reason you changed the call count from 2 to 3?

I actually ran this test locally, once with the value set to 3, again with the value set to 2, and again with the line removed entirely, and all three runs passed 🤔 

Might be best to just delete this line? I suspect counterfeiter returns empty values for calls with indexes that weren't specified ahead of time in stubs.